src/ResourceManager/TrafficManager/Commands.TrafficManager2/CustomHeader/AddAzureTrafficManagerCustomHeaderToEndpoint.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.TrafficManager.Models;
+using Microsoft.Azure.Commands.TrafficManager.Utilities;
+using System.Management.Automation;
+using ProjectResources = Microsoft.Azure.Commands.TrafficManager.Properties.Resources;
+
+namespace Microsoft.Azure.Commands.TrafficManager
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    [Cmdlet(VerbsCommon.Add, "AzureRmTrafficManagerCustomHeaderToEndpoint", SupportsShouldProcess = false), OutputType(typeof(TrafficManagerEndpoint))]
```

Comment:
All new cmdlets will need to have this cmdlet declaration:
```
[Cmdlet(VerbsCommon.Add, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "TrafficManagerCustomHeaderToEndpoint"
```
in order to support both Desktop and Netstandard.